sensor_msgs/NavSatFix start
sensor_msgs/NavSatFix end
---
geo_coord result
